interfaces/seccomp/template.go

```diff
@@ -200,7 +200,13 @@ inotify_rm_watch
 # input (man tty_ioctl), so we disallow it to prevent snaps plugging interfaces
 # with 'capability sys_admin' from interfering with other snaps or the
 # unconfined user's terminal.
+# similarly, TIOCLINUX allows to fake input as well (man ioctl_console) so
+# disallow that too
 # TODO: this should be scaled back even more
+~ioctl - TIOCSTI
+~ioctl - TIOCLINUX
+# restrict argument otherwise will match all uses of ioctl() and allow the rules
+# that were disallowed above - TODO: why does this still restrict TIOCLINUX?
 ioctl - !TIOCSTI
```

Comment:
Do not we just need `ioctl` without `!TIOCSTI`?

Reply:
Yep - updated in https://github.com/snapcore/snapd/pull/12849/commits/a46f902fcdfc7dab2534d98b2f452fd6df3c3806